src/lib/user-thumbnail.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @user-thumbnail
+ * Utility functions to return thumnail-related strings
+ */
+
+/**
+ * Generate a new empty sprite. The caller should provide localized versions of the default names.
```

Comment:
This looks copy-pasted from somewhere else.  But like I said above, this function probably doesn't belong in GUI since it's a detail of www.

Reply:
Oops, should have changed that line. For now, I moved this logic to www, though I hear you that it makes more sense for it to ultimately come from API.

Reply:
Done.